File: Data Structures & Algorithms/majority-element-ii/submission-3.java
class Solution {
    public List<Integer> majorityElement(int[] nums) {
        Set<Integer> res = new HashSet<>();
        for(int num : nums) {
            int cnt = 0;
            for(int i : nums) {
                if(i == num) cnt++;
            }
            if(cnt > nums.length / 3) {
                res.add(num);
            }
        }
        return new ArrayList<>(res);
    }
}

File: Data Structures & Algorithms/sort-colors/submission-3.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        vector<int> cnt(3);
        for(int num : nums) {
            cnt[num]++;
        }
        int idx = 0;
        for(int i = 0; i < 3; i++) {
            for(int j = 0; j < cnt[i]; j++) {
                nums[idx++] = i;
            }
        }
    }
};

// 0 1
// 1 2 
// 2 1